### DISCUSSION TITLE
Starting outline counter from 0

### DISCUSSION THREAD

**Discussion by roaldarbol** [typst/typst#4054] [Q&A] [answered]
I'm trying to figure out how to start the outline counter from a different value than 1 (0 would be useful), but have no been able to figure it out. Does anyone have any idea how to achieve that?

**Comment by roaldarbol** [comment]
As a prelude, it's part of a bigger issue which might be harder: I have some docs that should go before the ToC, and preferably receive roman numerals in the ToC. And then after the ToC comes all the normal content for which the ToC should start at 1 and continue - s the ToC needs to change numbering method throughout, which I also have been unable to figure out how to do. 
My current workaround would then be to just start from 0, so the actual chapters start from 1.

**Reply by l0wx** [reply, marked as answer]
```typst
#set heading(numbering: "I.1")

= Prelude
== Sub-Prelude
#lorem(99)

#outline()
#context counter(heading).update(0)
#set heading(numbering: "1.1")

= Gist
== Sub-Gist
#lorem(99)
```
